src/fr/hds/src/eu/kanade/tachiyomi/animeextension/fr/hds/Hds.kt

```diff
@@ -0,0 +1,75 @@
+package eu.kanade.tachiyomi.animeextension.fr.hds
+
+import eu.kanade.tachiyomi.animesource.model.SAnime
+import eu.kanade.tachiyomi.animesource.model.Video
+import eu.kanade.tachiyomi.lib.filemoonextractor.FilemoonExtractor
+import eu.kanade.tachiyomi.lib.streamhidevidextractor.StreamHideVidExtractor
+import eu.kanade.tachiyomi.multisrc.dooplay.DooPlay
+import eu.kanade.tachiyomi.network.GET
+import eu.kanade.tachiyomi.util.asJsoup
+import eu.kanade.tachiyomi.util.parallelCatchingFlatMapBlocking
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+import kotlinx.serialization.json.Json
+import okhttp3.Response
+import org.jsoup.nodes.Document
+import uy.kohesive.injekt.injectLazy
+
+class Hds : DooPlay(
+    "fr",
+    "HDS",
+    "https://www.hds.quest",
```

Comment:
This one should be the same as the one in build.gradle

Reply:
fixed in build.gradle